common/casus_belli_types/00_civil_war.txt

```diff
@@ -1200,58 +1200,9 @@ independence_war = {
 
```

Comment:
Also, I should say I wait for the history branch.
You have some kind of script, right?
So I think we should move many cultures a few eras forward. Like Gilneas and Kul Tiras should be in Early Modern (with riflemen, trade companies (Kul Tiras), production lines (Gilneas), but it will be weird if other humans will be in Early Medieval so we need to move them to Late Medieval(?)

Reply:
Okay I can do that first. I will double check in dev where every culture should be, but from what I'm seeing you want a lot of different tech tier cultures to be on early modern